src/analytics/Events.tsx

```diff
@@ -66,8 +66,11 @@ export enum SettingsEvents {
 
 export enum OnboardingEvents {
   onboarding_education_start = 'onboarding_education_start',
+  onboarding_education_start_experiment = 'onboarding_education_start_experiment',
   onboarding_education_scroll = 'onboarding_education_scroll',
+  onboarding_education_screen_impression = 'onboarding_education_screen_impression',
```

Comment:
nit: maybe?
```suggestion
  onboarding_education_step_impression = 'onboarding_education_step_impression',
```

Also does this fire for the initial step?

Reply:
Fixed in 0465ae5f7b225fba739040f340e0d6909b695709!